appinfo/info.xml

```diff
@@ -2,7 +2,7 @@
 <info>
 	<id>richdocuments</id>
 	<name>Collabora Online</name>
-	<description>An ownCloud app to work with office documents</description>
+	<description>Collabora Online allows you to to work with all kind office documents interactively direct in your browser.</description>
 	<licence>AGPL</licence>
 	<version>@PACKAGE_VERSION@</version>
```

Comment:
@PACKAGE_VERSION@ will be replaced during the build process as far as I can tell.
A simple rename will not work out